src/common/controllers.lua

```diff
@@ -18,9 +19,10 @@ end
 
 --- Remove a controller.
 ---@param controller string The controller to remove.
+---@param allowUnsafeAddresses boolean Whether to allow unsafe addresses
 ---@return string[]
-function controllers.removeController(controller)
-	utils.validateArweaveId(controller)
+function controllers.removeController(controller, allowUnsafeAddresses)
+	assert(utils.isValidAOAddress(controller, allowUnsafeAddresses), "Invalid AO Address")
 	local controllerExists = false
 
```

Comment:
I'd imagine that we don't need to assert addresses on remove controller. If they're on the ANT as a controller with any address, the owner should be able to remove them

Reply:
agreed

Reply:
[a44924e](https://github.com/ar-io/ar-io-ant-process/pull/38/commits/a44924e9d1952b43bdc8ed53222edcb3d3403087)

I did assert at a minimum it must be a string